react/tic-tac-toe/src/app/screens/Signup/styles.module.scss

```diff
@@ -0,0 +1,59 @@
+@import 'colors';
```

Comment:
You should have an applications.scss where you import all of your generic styles. 

This file should be in the following address: `src/scss/applications.scss`